activerecord/lib/active_record/migration.rb

```diff
@@ -140,6 +140,7 @@ def initialize(message = nil)
 
   class ConcurrentMigrationError < MigrationError #:nodoc:
     DEFAULT_MESSAGE = "Cannot run migrations because another migration process is currently running.".freeze
+    RELEASE_LOCK_FAILED_MESSAGE = "Failed to release advisory lock".freeze
```

Comment:
Can we just inline this? We don't need to move these to constants after Ruby 2.3

Reply:
We could but I used a constant so that I can test for the right message in our tests. Alternatively, would you prefer if we use a different name for the error instead?